addons/website_sale/tests/test_website_editor.py

```diff
@@ -5,6 +5,7 @@
 from odoo.addons.website.tools import MockRequest
 from odoo.exceptions import ValidationError
 from odoo.tests import HttpCase, tagged
+from odoo import Command
```

Comment:
First to keep alphabetical order?

Reply:
Oops. My logic was copied from other place and I though "Ok since it is from Odoo only (not a sub module), it can be at the end, but it does not make sense.